code/modules/detectivework/microscope/_forensic_machine.dm

```diff
@@ -0,0 +1,109 @@
+/obj/machinery/forensic
+	name = "forensic omnianalyzer"
+	desc = "A highly advanced microscope capable of analyzing all sorts of forensic evidence."
+	icon = 'icons/obj/forensics.dmi'
+	icon_state = "microscope"
+	anchored = 1
+	density = 1
+	var/obj/item/sample
+	var/report_num
+	var/list/allowed_evidence_types = list(
+		/datum/forensics/fingerprints,
+		/datum/forensics/fibers,
+		/datum/forensics/gunshot_residue,
+		/datum/forensics/trace_dna,
+		/datum/forensics/blood_dna
+	)
+
+/obj/machinery/forensic/Destroy()
+	if(sample)
+		sample.dropInto(loc)
```

Comment:
Needs to be nulled, also needs to return ..().

Reply:
done